clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/DataRecoveryClient.java

```diff
@@ -15,19 +16,25 @@
 public class DataRecoveryClient {
   private static final Logger LOGGER = LogManager.getLogger(DataRecoveryClient.class);
   private final DataRecoveryExecutor executor;
+  private final PlanningExecutor _planningExecutor;
```

Comment:
If the command is named 'estimate-data-recovery-time', Would it be more consistent to rename it to `Estimator` and  `estimator`?

Reply:
renamed :)